From 3db795bc08db735a719917637b33da8bbfd11b9b Mon Sep 17 00:00:00 2001
From: Sergiy Kibrik <sakib@meta.ua>
Date: Tue, 1 Oct 2013 15:35:39 +0300
Subject: [PATCH] use relative path for languages search

Search locales under DATADIR, which allows to install library under different prefixes.
---
 src/Makefile.am   |    2 +-
 src/liblanguage.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index 98bcc42..24ee0bf 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -2,7 +2,7 @@ lib_LTLIBRARIES = liblanguage.la
 liblanguage_la_SOURCES = liblanguage.c
 include_HEADERS = liblanguage.h
 
-AM_CFLAGS = -std=gnu99 -Wall
+AM_CFLAGS = -std=gnu99 -Wall -DDATADIR='"$(datadir)"'
 
 INCLUDES = @LIBLOPS_CFLAGS@
 liblanguage_la_LDFLAGS = @LIBLOPS_LIBS@ -no-undefined -version-info 1:0:0
diff --git a/src/liblanguage.c b/src/liblanguage.c
index f1f7f23..79ddf18 100644
--- a/src/liblanguage.c
+++ b/src/liblanguage.c
@@ -30,7 +30,7 @@
 #include <stdlib.h>
 #include "liblops.h"
 
-#define LOCALE_DIR "/usr/share/i18n/languages"
+#define LOCALE_DIR DATADIR "/i18n/languages"
 
 #define MAXPATH 256
 #define BUFSIZE 256
-- 
1.7.9.5

